UN-OCHA/unocha-site
# [UNO-651] Fix webp

Refs: UNO-651

This PR adds some patches for the `imageapi_optimize_webp` module to properly generate derivatives for `webp` images and to use `imagemagick` instead of `gd`.

This also includes a change to the nginx drupal.conf to restore checking if the source image exists for webp. This works because `imageapi_optimize_webp` preserves the original file extension when generating webp derivatives.

### Tests
 
1. Checkout the branch
2. Recreate the image
3. Recreate the container
4. Clear the cache, import the config
5. Go to `/admin/content/media` add a non webp image, check that the thumbnail appears
6. Go to `/admin/content/media` add a webp image, check that the thumbnail appears
